tools/cleed_io/reader.py

```
@@ -0,0 +1,114 @@
+        Atoms: ASE Atoms object containing Bulk + Overlayer.
+    """
+    
+    # 1. Parse Bulk File
+    a1 = a2 = a3 = None
+    bulk_atoms = []
+    bulk_symbols = []
+    
+    with open(bul_file, 'r') as f:
+        for line in f:
+            line = line.strip()
+            if not line or line.startswith('#') or line.startswith('c:'):
```

Ruff (0.14.8)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)